examples/protection_policies_v2/protection_policies_v2.yml

```diff
@@ -0,0 +1,1187 @@
+# Summary:
+# This playbook will do the following:
+# 1. Create a cluster for promoting VG
+# 2. Create categories for protection policies
```

Comment:
Have this examples in three separate examples:
1. Protection policies CRUD
2. VM as protected resource and it ops
3. VG as protected resource and it ops

Reply:
Assume that you have prerequisites.

Reply:
Have specific examples rather than clustered one big example